### PR TITLE
Enable workflows (4.13 GH) + Update ODF operator version (4.13 CI)

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
https://issues.redhat.com/browse/RHSTOR-4418

### CODE REVIEW

**Review by bipuladh — COMMENTED** [review]
Please squash the commits.

File: openshift-ci/odf-catalog-source.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ spec:
   icon:
     base64data: ''
     mediatype: ''
-  image: quay.io/rhceph-dev/ocs-registry:latest-stable-4.12.0
+  image: quay.io/rhceph-dev/ocs-registry:latest-stable-4.13.0
```
Comment (SanjalKatiyar) [inline openshift-ci/odf-catalog-source.yaml:31]: update "channel" in the Subscription yaml...

**Pull request merged**